backend/authentication/__init__.py

```diff
@@ -0,0 +1 @@
+"""Initialize the authentication package."""
```

Comment:
From there it'd be great if we can convert the `__init__.py` files back to empty files.